exercises/practice/equilateral_triangle/.docs/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Use Z3 to implement a function that, when given two points on the 2D coordinate 
```

Comment:
I don't think that you should say "Use Z3 to ..." since this will be under the Z3 track. Maybe just start the sentence with "Implement ..."